### DISCUSSION TITLE
Support for OpenAPI / Swagger UI

### DISCUSSION THREAD

**Discussion by gavlinski** [hbstack/.github#65] [Ideas]
Hi! Is there a support for a shortcode like [OpenAPI](https://mcshelby.github.io/hugo-theme-relearn/shortcodes/openapi/index.html) and [Swagger UI](https://github.com/swagger-api/swagger-ui) library?

**Comment by razonyang** [comment]
Not yet, will see how to implement it.

**Comment by razonyang** [comment]
Done, please check out the [doc](https://hugomods.com/docs/swagger-ui/) for using it.

**Comment by razonyang** [comment]
Edited, please use `github.com/hbstack/swagger-ui` instead, which applies extra styles to match dark mode of HB themes.

**Comment by gavlinski** [comment]
Thank you very much! I'll test it in the next days. :+1:

**Reply by razonyang** [reply]
Btw, the dark mode not play well until it was [supported officially](https://github.com/swagger-api/swagger-ui/issues/5327).
